examples/core/transform-feedback/package.json

```diff
@@ -7,11 +7,11 @@
     "start-local": "webpack-dev-server --env.local --progress --hot --open"
   },
   "dependencies": {
-    "luma.gl": "^4.0.0"
+    "luma.gl": ">=4.0.0-alpha.6"
```

Comment:
Good change! We should do this for every examples in luma.gl and deck.gl, and the frameworks themselves. This has been mentioned by @heshan0131